detectron2_detector/config/detectron2.yaml

```diff
@@ -0,0 +1,13 @@
+detectron2_node:
+  ros__parameters:
+    # detectron parameters
+    detectron_config_file: "COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml"
+    detectron_score_thresh: 0.8
+    # processing parameters
+    pointcloud2_topic: "/camera/depth/points"
+    categories: [0] # please check out COCO dataset category_id list if you want to config this; if you want to track all, leave it empty
```

Comment:
So is this by line number in the COCO file or by string? If by line #, mention that in the readme. That's probably obvious to AI engineers but probably not for robotics engineers